Vagrantfile

```diff
@@ -0,0 +1,35 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box_url = "https://download.fedoraproject.org/pub/fedora/linux/releases/23/Cloud/x86_64/Images/Fedora-Cloud-Base-Vagrant-23-20151030.x86_64.vagrant-libvirt.box"
+  config.vm.box = "f23-cloud-libvirt"
+  config.vm.network "forwarded_port", guest: 5000, host: 5001
+  config.vm.synced_folder ".", "/vagrant", type: "sshfs"
+
+  config.vm.provision "shell", inline: "sudo dnf -y install python redhat-rpm-config python-devel postgresql-devel postgresql-server rpl python-alembic python-psycopg2 gcc"
+
+  config.vm.provision "shell", inline: "pip install kitchen paver urllib3"
+  config.vm.provision "shell", inline: "pip install -r /vagrant/requirements.txt"
+
+  config.vm.provision "shell", inline: "sudo postgresql-setup initdb"
+
+  config.vm.provision "shell", inline: "sudo rpl 'host    all             all             127.0.0.1/32            ident' 'host    all             all             127.0.0.1/32            trust' /var/lib/pgsql/data/pg_hba.conf"
+  config.vm.provision "shell", inline: "sudo rpl 'host    all             all             ::1/128                 ident' 'host    all             all             ::1/128                 trust' /var/lib/pgsql/data/pg_hba.conf"
```

Comment:
Thought about using sed instead of rpl? (just one less package to install)

But not a blocker :)